plasmapy/examples/basic_simulation.py

```diff
@@ -53,24 +54,25 @@ def riemann_shock():
     riemann.energy = energy
     print('Success')
 
-    # for maxt in np.arange(0, 0.21, 0.02):
-    #     print('- Running Simulation... ', end='')
-    #     riemann.simulate(max_time=maxt*u.s)
-    #     print("Simulation complete")
-
-    #     fig, ax = plt.subplots(2, 2)
-    #     ax = ax.flatten()
-    #     ax[0].plot(x, riemann.density)
-    #     ax[0].set_ylim(0, 1.2)
-    #     ax[1].plot(x, riemann.velocity[0, :, ...])
-    #     ax[1].set_ylim(0, 1.1)
-    #     ax[2].plot(x, riemann.energy)
-    #     ax[3].plot(x, riemann.pressure)
-    #     ax[3].set_ylim(0, 1.2)
-    #     plt.savefig(
-    #         str(savedir/"riemann_shock_{:.4f}".format(maxt)).replace('.', '_'))
-    #     plt.close()
     t0 = datetime.datetime.now()
+    for maxt in np.arange(0, 0.21, 0.02):
+        print('- Running Simulation... ', end='')
+        riemann.simulate(max_time=maxt*u.s)
+        print("Simulation complete")
+
+        fig, ax = plt.subplots(2, 2)
```

Comment:
I always liked doing it like this:

    fig, ((density_ax, velocity_ax), (energy_ax, pressure_ax)) = plt.subplots...